ts/src/bitteam.ts

```diff
@@ -0,0 +1,2395 @@
+        const currencies = this.safeValue (responseResult, 'currencies', []);
+        // usding another endpoint to fetch statuses of deposits and withdrawals
+        let statusesResponse = await this.publicGetTradeApiCmcAssets ();
+        //
+        //     {
+        //         "ZNX": {
+        //             "name": "ZeNeX Coin",
+        //             "unified_cryptoasset_id": 30,
+        //             "withdrawStatus": true,
+        //             "depositStatus": true,
+        //             "min_withdraw": 0.00001,
+        //             "max_withdraw": 10000
+        //         },
+        //         "USDT": {
+        //             "name": "Tether USD",
+        //             "unified_cryptoasset_id": 3,
+        //             "withdrawStatus": true,
+        //             "depositStatus": true,
+        //             "min_withdraw": 1,
+        //             "max_withdraw": 100000
+        //         },
+        //     }
+        //
+        statusesResponse = this.indexBy (statusesResponse, 'unified_cryptoasset_id');
+        const result = {};
+        for (let i = 0; i < currencies.length; i++) {
+            const currency = currencies[i];
+            const id = this.safeString (currency, 'symbol');
+            const numericId = this.safeInteger (currency, 'id');
+            const code = this.safeCurrencyCode (id);
+            const active = this.safeValue (currency, 'active', false);
+            const precision = this.safeInteger (currency, 'precision');
+            const txLimits = this.safeValue (currency, 'txLimits', {});
+            const minWithdraw = this.safeString (txLimits, 'minWithdraw');
+            const maxWithdraw = this.safeString (txLimits, 'maxWithdraw');
+            const minDeposit = this.safeString (txLimits, 'minDeposit');
+            let fee = undefined;
+            const withdrawCommissionFixed = this.safeValue (txLimits, 'withdrawCommissionFixed', {}) as any;
+            let feesByNetworkId = {};
+            const blockChain = this.safeString (currency, 'blockChain');
+            // if only one blockChain
+            if ((blockChain !== undefined) && (blockChain !== '')) {
+                fee = this.parseNumber (withdrawCommissionFixed);
+                feesByNetworkId[blockChain] = fee;
+            } else {
+                feesByNetworkId = withdrawCommissionFixed;
+            }
+            const statuses = this.safeValue (statusesResponse, numericId, {});
+            const deposit = this.safeValue (statuses, 'depositStatus');
+            const withdraw = this.safeValue (statuses, 'withdrawStatus');
+            const networkIds = Object.keys (feesByNetworkId);
+            const networks = {};
+            const networkPrecision = this.safeInteger (currency, 'decimals');
+            for (let j = 0; j < networkIds.length; j++) {
+                const networkId = networkIds[j];
+                const networkCode = this.networkIdToCode (networkId);
```

Comment:
whenever available, we should provide the currnecy to `networkIdToCode/networkCodeToId` 
eg: this.networkIdToCode(networkId, currencyCode)`